### ISSUE TITLE
copyright file in packages not correct

### ISSUE THREAD

**Issue by ghost** [eduvpn/eduvpn-debian#11]
the VPN components themselves are AGPLv3+ and not MIT.

**Comment by joostvb-gh** [comment]
Hi,

I've checked all 20 sources for copyright statement, and compared with what's in debian/copyright.  I've found no problems in:

auth-script-openvpn BaconQrCode php-json-signer php-oauth2-client php-oauth2-server php-otp-verif
ier php-secookie php-sqlite-migrate php-yubitwee Twig-extensions

After my PR got merged, there are no problems in:

vpn-admin-portal vpn-lib-common vpn-server-api vpn-server-node vpn-user-portal

I've reported https://github.com/eduvpn/vpn-portal-artwork/issues/1 describing problems with

php-saml-ds-artwork-eduVPN  vpn-portal-artwork-eduVPN vpn-portal-artwork-LC

Changes are likely needed for:

php-openvpn-connection-manager php-saml-ds

.

php-saml-ds might need full quote of Apache license text, see auth-script-openvpn for example.

php-openvpn-connection-manager is licensed under GNU AGPL v3 : "Copyright: 2016-2018, The Commons Conservancy eduVPN Programme", it lacks license information for debian/*.

I'll create another PR to deal with these last 2 packages.

Bye, Joost
